Makefile

```diff
@@ -27,6 +27,15 @@ fix:
 
 lint-fix: fix
 
+migrate: build
+	npx knex migrate:latest --knexfile './dist/knexfile.ts'
+
+migrate-rollback: build
+	npx knex migrate:rollback --knexfile './dist/knexfile.ts'
+
+migration: build
+	npx knex migrate:make $(name) -x ts --knexfile './dist/knexfile.ts'
```

Comment:
```suggestion
```

Lets just remove this? It's less useful i think, and the name might confuse people